### PR TITLE
feat(auto-balance): disk in pressure

### PR DESCRIPTION
**Pull request by c3y1huang**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#4105

#### What this PR does / why we need it:

Introduce new fields in the proto request message for proxying the added replica to use file local sync.

#### Special notes for your reviewer:

`None`

#### Additional documentation or context

`None`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,6 +2,8 @@ module github.com/longhorn/longhorn-instance-manager
 
 go 1.21
 
+replace github.com/longhorn/longhorn-engine => github.com/c3y1huang/longhorn-engine v1.6.0-dev-20240105.0.20240201041530-b0e5b9da243c
```
Comment (c3y1huang) [inline go.mod:5]: TODO: update after https://github.com/longhorn/longhorn-engine/pull/1003.

**Comment by innobead** [comment]
Is this ready?

**Comment by c3y1huang** [comment]
@derekbit this is ready for review.

**Review by derekbit — APPROVED** [review]
LGTM

**Pull request merged**